python-package/lightgbm/callback.py

```diff
@@ -73,15 +73,13 @@ class CallbackEnv:
 
 def _format_eval_result(value: _EvalResultTuple, show_stdv: bool) -> str:
     """Format metric string."""
-    if len(value) == 4:
-        return f"{value[0]}'s {value[1]}: {value[2]:g}"
-    elif len(value) == 5:
-        if show_stdv:
-            return f"{value[0]}'s {value[1]}: {value[2]:g} + {value[4]:g}"  # type: ignore[misc]
-        else:
-            return f"{value[0]}'s {value[1]}: {value[2]:g}"
-    else:
-        raise ValueError("Wrong metric value")
+    eval_name, metric_name, eval_result, *_ = value
```

Comment:
What do you think of using similar names to the ones here https://github.com/microsoft/LightGBM/blob/53e0ddf7cd6eb281e3bec6273b19ff541c69bfa6/python-package/lightgbm/basic.py#L5215
I find the first one a bit confusing, since it's the name of the validation set. Maybe something like `dataset_name, metric_name, metric_value, *_ = value`?

Reply:
You're totally right... I think `eval_name` and `eval_result` are confusing. I love your suggested names, agree we should standardize on those (here and in future PRs like this).

Reply:
updated in https://github.com/microsoft/LightGBM/pull/6749/commits/6b63377a57fb013f063bc5a3be2ce278e183c95a

I'll do more of this renaming in future PRs.